addons/mail/models/mail_thread.py

```diff
@@ -4507,9 +4507,10 @@ def _message_update_content(self, message, body, attachment_ids=None, partner_id
         if strict:
             self._check_can_update_message_content(message.sudo())
 
-        msg_values = {
-            'body': escape(body),  # keep html if already Markup, otherwise escape
-        } if body is not None else {}
+        msg_values = {}
+        if body is not None:
+            # keep html if already Markup, otherwise escape
+            msg_values["body"] = escape(body) + Markup("<span class='o-mail-message-edited'/>")
```

Comment:
Not a huge deal, but a `<span>` by itself you have some UI presence, e.g. some padding/margin/width/height. It could also affect subtly affect CSS styles that target `span`.

Former issue could be solved with setting `display: none`, later is more complicated thou thankfully rarely an issue, custom tag would do it but it's not allowed by our HTML sanitizer.